### PR TITLE
Add entities and blips

### PR DESCRIPTION
**Pull request by arufanov**
Add entities and blips

### CODE REVIEW

File: src/main/resources/database/data/blips_en.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title|description|created_by|last_modified_by
+Languages|Programming languages we use|tabr|tabr
```
Comment (yurichervoniy) [inline src/main/resources/database/data/blips_en.csv:2]: Isn't it a segment?

File: src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml [hunk 2]
```diff
@@ -33,11 +33,6 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
-        <!-- current Liquibase ignores checkConstraint attribute -->
-        <sql>
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml:37]: Are we sure about it?

Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml]: @yurichervoniy 
back it.

File: src/main/resources/database/data/blips_en.csv (hunk 1 above)
Comment (arufanov) [reply src/main/resources/database/data/blips_en.csv]: @yurichervoniy 
remove file, will use wizard on initial start up to populate data.

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <include file="../properties.yml" relativeToChangelogFile="true"/>
+
+    <changeSet id="create_entities_table" author="tabr">
+        <createTable tableName="entities" remarks="Holds information about entities.">
+            <column name="id" type="integer" remarks="Segment id, primary key." autoIncrement="true"
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml:11]: Segment id -> Entity id

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                    generationType="ALWAYS">
+                <constraints primaryKey="true" primaryKeyName="pk_entities_id" nullable="false"/>
+            </column>
+            <column name="title" type="${text.type}" remarks="Segment title, unique.">
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml:15]: Segment -> Entity

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                <constraints unique="true" uniqueConstraintName="uc_entities_title" nullable="false"/>
+            </column>
+            <column name="description" type="${text.type}" remarks="Description for segment."/>
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml:18]: segment -> entity

File: src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml [hunk 6]
```diff
@@ -7,28 +7,28 @@
...
                 <constraints primaryKey="true" primaryKeyName="pk_radars_id" nullable="false"/>
             </column>
-            <column name="title" type="${text.type}" remarks="Radar title">
+            <column name="title" type="${text.type}" remarks="Radar title, unique.">
                 <constraints unique="true" uniqueConstraintName="uc_radars_title" nullable="false"/>
             </column>
-            <column name="description" type="${text.type}" remarks="Description for Radar"/>
-            <column name="is_default" type="boolean" remarks="Whenever the radar is default" defaultValueBoolean="false">
+            <column name="description" type="${text.type}" remarks="Description for radar."/>
+            <column name="is_default" type="boolean" remarks="Whenever the radar is default." defaultValueBoolean="false">
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml:19]: I dont like such checks 
You can easilly get in situation where you have multiple rows with is_default=true;

I prefer separate table or trick with null value and unique index. Only one row can have a value, others null. And create unique index (null values is not indexed)

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+                    defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <!-- current Liquibase ignores checkConstraint attribute -->
+        <sql>
+            ALTER TABLE entities
+                ADD CONSTRAINT ck_entities_title_trim CHECK (trim(title) = title)
+        </sql>
+    </changeSet>
```
Comment (yurichervoniy) [inline src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml:41]: Where we will store information abou versions?

File: src/main/resources/database/data/blips_en.csv (hunk 1 above)
Comment (arufanov) [reply src/main/resources/database/data/blips_en.csv]: @yurichervoniy 
* ?

File: src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml (hunk 2 above)
Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml]: @yurichervoniy 
?

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml (hunk 3 above)
Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml]: @yurichervoniy 
* fixed

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml (hunk 4 above)
Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml]: @yurichervoniy 
* fixed

File: src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml (hunk 6 above)
Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_10_0000_create_radars_table.xml:19]: @yurichervoniy 
* "ou can easilly get in situation where you have multiple rows with is_default=true;" it is not fatal. We will use the first one.
* "And create unique index (null values is not indexed)" I don't like this tricks, You will need (or not) extend table, for example add tenats, add kind of radar etc. DDL is not good.

File: src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml (hunk 7 above)
Comment (arufanov) [reply src/main/resources/database/migrations/2022_12_03_0000_create_entities_table.xml:41]: @yurichervoniy 
* I dont' know and don't care:) Current requirements: make as zolondo. The Zolando doesn't have version. So I don't care. 
* There is at leas one way to do it without refactoring schema (only extend): add a fk to list of available version to radar.  I support it isn't the best way, but before we need clarify requirements.  It is out of scope pr.

**Review by yurichervoniy — APPROVED** [review]

**Pull request merged**